packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -760,6 +799,11 @@ export class Project extends ProjectBase {
     return ret;
   }
 
+  private set encryptionKey(encryptionKey: kms.IKey) {
```

Comment:
Move the into `bindToCodePipeline`. This is not something we expect other private logic to implement.

#keephackslocal

Reply:
This setter is used twice (once in the constructor, and another time in `bindToCodePipeline`). If I moved it to `bindToCodePipeline`, this logic would have to be duplicated.

Does it make sense to leave it then?

Reply:
Okay yes makes sense.